transformer_engine/common/include/transformer_engine/transpose.h

```diff
@@ -90,8 +90,8 @@ void nvte_fp8_transpose_dbias(const NVTETensor input, NVTETensor transposed_outp
  *                                     transposed.
  *  \param[in]     stream              CUDA stream used for the operation.
  */
-void nvte_multi_cast_transpose(size_t num_tensors, const NVTETensor* input_list,
-                               NVTETensor* output_list, cudaStream_t stream);
+void nvte_multi_quantize(size_t num_tensors, const NVTETensor* input_list, NVTETensor* output_list,
```

Comment:
Can we still keep both functions and deprecate the old interface?

Reply:
I'm not sure if I understand your question. Do you mean keeping `nvte_multi_cast_transpose` (with the equivalent behavior of `nvte_multi_quantize`) but marking it as deprecated?

I changed the name because the transpose is now optional, so naming it `*_cast_transpose` might be a bit confusing.

Reply:
This change was to let delayed scaling on Blackwell also go to this fused `multi_cast_transpose` kernel, but as you mentioned, the performance may not be good. So I reverted it.